restful.js

```diff
@@ -160,7 +160,10 @@ module.exports = function setup(mount, vfs, mountOptions) {
           meta.stream.on("error", abort);
           if (options.encoding === null) {
             jsonEncoder(meta.stream, base, htmlBase).pipe(res);
-          } else {
+          } else if (mountOptions.responseWrapper){
```

Comment:
Should we check typeof mountOptions.responseWrapper === 'function' here?

Reply:
probably a good idea

Reply:
fixed!